sections/interfaces.js

```diff
@@ -368,9 +368,17 @@ export default [
     description: "Returns an array of query strings filled out with the correct values based on bindings, etc. Useful for debugging.",
     children: [
       {
-        type: "runnable",
+        type: "code",
+        language: "js",
         content: `
-          knex.select('*').from('users').where(knex.raw('id = ?', [1])).toString()
+          var toStringQuery = knex.select('*').from('users').where('id', 1).toString();
+          console.log(toStringQuery.toString());
```

Comment:
you are calling toString() for string here?

Reply:
If I remove the .toString() function call from ```toStringQuery```, will it still print out the query?